### PR TITLE
TeamH Wiki Update - Traffic Modelling Datasets

### PR DESCRIPTION
**Pull request by HobbySingh**

### CODE REVIEW

File: wiki/datasets/traffic-modelling-datasets.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Traffic modelling is a hot topic in the field of autonomous cars currently. Here you will find a list of open datasets which can be used as source for building a traffic model. The list will include data captured from a variety of sources as listed below:
+- UAV/Drones
+- Traffic Camera 
+- Autonomous Cars
+
+> This is not a list of datasets for learning-to-drive. This list is more focused towards dataset which provide global perspective of the traffic scenarios, rather than ego-vehicle perspective. Though few ego-vehicle datasets can be used for traffic modelling as well.
+
+Each dataset is scored based upon several parameters crucial for learning behavior and interactions between vehicles in a recorded scenario.
+
+<iframe height="100%" width="100%" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vRNtQVog9cMhYrOuv0fGlEKyK4HiuwD0orLoXEfO9z-Jtz9hvEB6qzdZy6lOSsyyPGakYe6ggYBAi8u/pubhtml?gid=0&amp;single=true&amp;widget=true&amp;headers=false"></iframe>
```
Comment (smbryan) [inline wiki/datasets/traffic-modelling-datasets.md:15]: This spreadsheet should be here and not linked to a Google doc. The spreadsheet also appears to be incomplete.

**Comment by HobbySingh** [comment]
> This spreadsheet should be here and not linked to a google doc. The spreadsheet also appears to be incomplete.

The table is broader and it is not possible to format that using Markdown. Hence I put that into a separate spreadsheet as it looks much better visually.

I have fixed the incomplete part.

**Comment by smbryan** [comment]
I applaud your ingenuity, but I can't accept adding that dependency. Markdown works just fine: https://kramdown.gettalong.org/quickref.html#tables

**Comment by HobbySingh** [comment]
I have committed the file that I already prepared with a table embedded in Markdown. You can look that the Markdown isn't flexible with large tables.

**Comment by HobbySingh** [comment]
Hi, I have updated the file. 

1. Reduced the table content so that it fits nicely in markdown and moved it as general description.
2. Added another dataset.

Please let me know if other changes are required.

**Comment by smbryan** [comment]
Looks good. Thanks!

**Pull request merged**